Gemfile

```diff
@@ -33,6 +33,8 @@ group :development, :test do
 
   gem 'puma'
 
+  gem 'haml'
```

Comment:
Hmm strange, always worked for me without this. I might have installed it manually on the system though.